ansible/roles/ocp4-workload-metering/tasks/workload.yml

```diff
@@ -6,29 +6,29 @@
     msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
 
 - name: Create OpenShift Objects for Metering
-  ignore_errors: yes
+  ignore_errors: true
   k8s:
     state: present
     merge_type:
     - strategic-merge
     - merge
     definition: "{{ lookup('template', item ) | from_yaml }}"
   loop:
-    - ./templates/metering_namespace.j2
-    - ./templates/operator_group.j2
-    - ./templates/catalog_source_config.j2
-    - ./templates/subscription.j2
+  - ./templates/metering_namespace.j2
+  - ./templates/operator_group.j2
+  - ./templates/catalog_source_config.j2
+  - ./templates/subscription.j2
```

Comment:
Let's take the opportunity to clean this up a bit more, `template` lookup already looks in `./templates/`:

```
  loop:
  - metering_namespace.j2
  - operator_group.j2
  - catalog_source_config.j2
  - subscription.j2
```